app/src/i18n/locales/pt/dashboard.json

```diff
@@ -205,5 +205,17 @@
   "yes-i-understand": "Sim, eu entendo",
   "account-frozen-warning-text": "A sua conta está congelada. Por favor, entre em contato com um administrador ou com <bold>{{email}}</bold> para reativá-la",
   "account-frozen-header": "Sua conta está congelada",
-  "multiple-sources": "Múltiplas fontes"
+  "multiple-sources": "Múltiplas fontes",
+  "km2": "km²",
+  "all-projects": "Todos os projetos",
+  "check-your-dashboard": "Verifique o seu painel de controle",
+  "check-your-dashboard-description": "Mantenha o controle das principais métricas da sua cidade em um só lugar",
+  "go-to-dashboard-cta": "Ir para o painel",
+  "tools-title": "As suas ferramentas",
+  "by": "Por {{author}}",
+  "launch": "Lançamento",
+  "no-modules-available": "Ainda não há módulos disponíveis",
+  "no-modules-description": "Você não tem nenhum módulo ativo para esta etapa! Entre em contato conosco para obter ajuda personalizada.",
+  "contact-us": "ENTRE EM CONTATO CONOSCO",
+  "journey": ""
```

Comment:
Missing translation object (nested objects not supported by our automatic translation tool)